packages/components/psammead-story-promo-list/CHANGELOG.md

```diff
@@ -3,9 +3,9 @@
 <!-- prettier-ignore -->
 | Version | Description |
 | ------- | ----------- |
-| 1.0.0 | [PR#741](https://github.com/BBC/psammead/pull/741) Remove alpha tag |
-| 1.0.0-alpha.1 | [PR#677](https://github.com/BBC/psammead/pull/677) Use `@bbc/gel-foundations@3.0.0` |
-| 1.0.0-alpha.0 | [PR#679](https://github.com/BBC/psammead/pull/679) Bump version number |
-| 0.1.0-alpha.3 | [PR#672](https://github.com/BBC/psammead/pull/672) Add margin to top of unordered list |
-| 0.1.0-alpha.2 | [PR#534](https://github.com/BBC/psammead/pull/534) Remove Timestamp padding. |
-| 0.1.0-alpha.1 | [PR#486](https://github.com/BBC/psammead/pull/486) Create initial package. |
+| 1.0.1   | [PR#713](https://github.com/bbc/psammead/pull/713) Update `styled-components` to 4.3.2 |
```

Comment:
This change has overwritten the 1.0.0 version, so we will need to bump again the `Story Promo List` to add another record in the `CHANGELOG`.